rackcity/utils/change_planner_utils.py

```diff
@@ -230,8 +244,11 @@ def get_many_assets_response_for_cp(request, change_plan, decommissioned=False,
     )
     if filter_failure_response:
         return filter_failure_response
+    if decommissioned:
+        asset_serializer = GetDecommissionedAssetSerializer(assets, many=True,)
```

Comment:
decommissioned assets need to use diff serializer

Reply:
so true.